src/ProfilesPage/ProfilePage.js

```diff
@@ -21,20 +24,23 @@ import EditPage from '../EditPage/EditPage'
 
 const userId = sessionStorage.getItem('userId');
 const adminUser = sessionStorage.getItem('adminUser');
+const admin = sessionStorage.getItem('isAdmin');
 // var editButtonClicked = false;
 
 class ProfilePage extends Component {
   componentWillMount() {
-
-    const sender_id = sessionStorage.getItem('userId');
-    const receiver_id = this.props.match.params.id;
-    const {fetchUserInfo, fetchUserFeeds, fetchUserConnections} = this.props;
-    this.props.fetchUserInfo(this.props.match.params.id);
-    this.props.fetchUserFeeds(this.props.match.params.id);
-    this.props.fetchUserConnections(this.props.match.params.id);
-    this.props.fetchConnectionStatus({sender_id, receiver_id});
-    this.setState({editUserEnable: false});
-  }
+      const sender_id = sessionStorage.getItem('userId');
+      const receiver_id = this.props.match.params.id;
+      const { fetchUserInfo, fetchUserFeeds, fetchUserConnections } = this.props;
+      this.props.fetchUserInfo(this.props.match.params.id);
```

Comment:
the purpose of destructuring in line 34 is that you don't need to write `this.props.fetchUserInfo(...`, instead `fetchUserInfo(...`

Reply:
please do the same for rest of method

Reply:
done